### PR TITLE
Applied many improvements

### PR DESCRIPTION
**Pull request by silvioprog**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,11 +57,17 @@ An example of how to use the library
   {#each countries as country}
     <li>{country.name}</li>
   {/each}
-  <SvelteInfiniteScroll threshold={100} on:loadMore={() => page++} />
+  <InfiniteScroll threshold={100} on:loadMore={() => page++} />
 </ul>
 ```
 
-### Another Examples
+In Sapper applications:
```
Comment (andrelmlins) [inline README.md:64]: The svelte property in package.json already solves this import issue. If possible, remove this message.

File: src/lib/InfiniteScroll.svelte [hunk 2]
```diff
@@ -90,4 +78,6 @@
   });
 </script>
 
-<div bind:this={component} style="width:0px" />
+{#if !window && !elementScroll}
+  <div bind:this={component} class="svelte-infinite-scroll" style="width: 0;" />
```
Comment (andrelmlins) [inline src/lib/InfiniteScroll.svelte:82]: what would be the reason for this class?

**Review by andrelmlins — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (silvioprog) [reply README.md]: Removed as suggested.

File: src/lib/InfiniteScroll.svelte (hunk 2 above)
Comment (silvioprog) [reply src/lib/InfiniteScroll.svelte]: It is just to identify the `svelte-infinite-scroll` component allowing to apply some external style to it. Now the class was replaced by an `id`. 🙂

**Review by andrelmlins — APPROVED** [review]

**Pull request merged**